common-rest/src/main/java/org/uniprot/api/rest/download/queue/EmbeddingsQueueConfigProperties.java

```diff
@@ -0,0 +1,23 @@
+package org.uniprot.api.rest.download.queue;
+
+import lombok.Data;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+@Data
+@ConfigurationProperties(prefix = "async.download.embeddings")
+public class EmbeddingsQueueConfigProperties {
```

Comment:
Embeddings should be in UniProtKB

Reply:
Yes, you are right. But i kept it in common so that queue related configs are at one place rather than scattered. If you still feel we should move it to uniprotkb-rest, i can move it.

Reply:
The problem of having it in common, is that these Bean will be loaded for every module that has asyncDownload profile (uniprot now, but idmapping, uniparc and uniref in a near feature), and it should only be instantiated for uniprotkb. Does it make sense to you?

Reply:
Yes, i agree. thats a good point. Moved to uniprotkb-rest